### ISSUE TITLE
Why use the" y_eval" of the tuning parameter as the test set for evaluation, shouldn't it be separate?

### ISSUE THREAD

**Issue by maymodric** [google/lifetime_value#12]

**Comment by Ty4Code** [comment]
I totally agree, it is likely not a good idea in general to use the test set as our 'validation_data' input for the TensorFlow fit function.

However, in this particular case, I don't think it's an issue because there is no early stopping and the validation loss is not used for any hyperparameter tuning.

In other words, if we were to remove the `validation_data=(x_eval, y_eval)` line from the notebook, it would not have any impact on the final results of the training run/notebook run as far as I can tell.
